### DISCUSSION TITLE
哪位大佬知道 home readme url 选项被我删了 怎么恢复

### DISCUSSION THREAD

**Discussion by kissyouhunter** [AlistGo/alist#1122] [Q&A]
手欠 点了一下 home readme url 旁边的删除按钮
然后 home readme url 就没有了
不知道怎么找回来

**Comment by anwen-anyi** [comment]
> 手欠 点了一下 home readme url 旁边的删除按钮
> 
> 然后 home readme url 就没有了
> 
> 不知道怎么找回来

没事删了就删了吧 那个没用了 现在有个新的全局的Readme url用那个就行

**Reply by kissyouhunter** [reply]
谢谢大佬
